recipes/bdbag/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "bdbag" %}
+{% set version = "1.5.6" %}
+{% set hash_type = "sha256" %}
```

Comment:
Not really important, but I'd probably just inline this. It's a bit nonstandard to push this into a variable I believe.

```suggestion
```

Reply:
Same thing with `hash_value`

Reply:
True. I suppose that the automatic updates (regro-cf-autotick-bot) won't update the SHA256 automatically for you if you are using any variables there.